pytorch_lightning/loggers/wandb.py

```diff
@@ -56,6 +56,8 @@ class WandbLogger(LightningLoggerBase):
 
     Example:
 
+    .. code-block:: python
```

Comment:
```suggestion
    .. code::
```
can we have it as tested code, would it work?

Reply:
You mean as part of `test_wandb.py`?
I can try, we may not need the `mock` any longer and try to run wandb in `drymode` which would create the files locally (and may be slightly longer).

It's not a super easy task (for me) so would you be ok to create a separate issue for it and assign it to me?